iot/api/CloudIotTest/test/data/ec_public.pem

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAES5Mo86JNproHuRgdDxwadgXcMB9L
```

Comment:
Instead of pushing the changelist with sample cert, should we rather put instructions on how to create a sample cert as part of the demo setup?

Reply:
This is just used for testing, the README.md in the folder should deter folks from registering devices (they would be unusable without the private keys).